packages/cli/src/cog/job.ts

```diff
@@ -134,9 +126,17 @@ export const CogJobFactory = {
             logger.info(
                 {
                     // Size of the biggest image
-                    big: getTileSize(firstQk, metadata.resolution),
+                    big: ctx.targetProjection.getImageWidth(
+                        metadata.resZoom,
+                        firstQk.length,
+                        ctx.targetProjection.tms.tileSize * 1.5,
```

Comment:
We generate `BLOCKSIZE` sized tiles not `tileSize`

Reply:
Do we want the BLOCKSIZE hard coded? Do we want it the same for all Epsg codes?

Reply:
This is where GDAL math might come in handy, it lets us have a more fine grained choice of target resolution. 

I believe we are likely wanting to have a bigger block size, 512, 1024 or 2048 because it gives a better image quality. but we will want to calculate which one based on image vs tms pixel scales.